lib/Default/SmrWeapon.class.php

```diff
@@ -153,7 +153,8 @@ public function getModifiedAccuracyAgainstPlayer(AbstractSmrPlayer $weaponPlayer
 		$weaponShip = $weaponPlayer->getShip();
 		$targetShip = $targetPlayer->getShip();
 		$mrDiff = $targetShip->getMR() - $weaponShip->getMR();
-		if ($mrDiff > 0) {
+		//the 500 cap is to exclude huge numbers generated by the SMS which can cause accuracy to become absolute
```

Comment:
I think it'd make more sense to do
```php
$mrDiff = min($mrDiff, 500); // Cap the mr diff to 500
```

Reply:
That is a scalpel to my chain saw.  I like it.

That will result in a 500MR difference instead of ignoring MR entirely.  A 500 MR difference will result in accuracy being degraded by 33% when a SMS fires upon a standard ship.

At second glance, I believe it would be better if negative MR values weren't returned in the first place.  I'm going to update this accordingly.